docs/mosviz/redshift.rst

```diff
@@ -16,3 +16,6 @@ using :meth:`~jdaviz.configs.mosviz.helper.Mosviz.update_column`.
 The 1D and 2D spectrum objects can be retrieved (with redshift optionally applied) using
 :meth:`~jdaviz.configs.mosviz.helper.Mosviz.get_spectrum_1d` and :meth:`~jdaviz.configs.mosviz.helper.Mosviz.get_spectrum_2d`,
 respectively.
+
+See the ``MosvizNIRISSExample.ipynb`` notebook in the repository to see examples of manipulating
```

Comment:
I think we should either make the notebook name or "repository" a link to the appropriate location, just to make navigating over to Github as easy as possible for someone reading the docs.

Reply:
should I just link to https://github.com/spacetelescope/jdaviz/tree/main/notebooks?

Reply:
Sounds good to me.